### PR TITLE
make rank not null

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #140** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/140" target="_blank">Graphite</a> 
    * **PR #141** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141" target="_blank">Graphite</a> 
      * **PR #142** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/142" target="_blank">Graphite</a> 
        * **PR #143** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/143" target="_blank">Graphite</a> 
          * **PR #145** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145?utm_source=stack-comment).

File: subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt [hunk 1]
```diff
@@ -19,11 +19,13 @@ class GetMyRankUseCase(
 
         if (accessibilityRank == null) {
             val user = userApplicationService.getUser(userId) ?: throw SccDomainException("잘못된 계정입니다.")
+            // if lastRank can not be found, then the user is the first rank
```
Comment (Zeniuus) [inline subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: 이게 무슨 뜻이에요?

**Review by Zeniuus — APPROVED** [review]
대충 아무 값이나 넣는다고 이해했습니당

File: subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt (hunk 1 above)
Comment (bellatoris) [reply subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: conquest가 0인 rank 를 찾을 수 없다는 건 updateRank를 한번도 안돌렸다는거라고 생각해서 걍 1등으로 넣어도 된다고 생각했어요 ㅋㅋ

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: 한 번도 안 돌린 유저가 무조건 있겠죠...? ㅋㅋ

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: 내가 그런 유저다

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: 가라 유저 하나 만들어두면 되겠네 ㅋㅋㅋ

Comment (bellatoris) [reply subprojects/bounded_context/accessibility/application/src/main/kotlin/club/staircrusher/accessibility/application/port/in/GetMyRankUseCase.kt:22]: 무조건 있지 않을가요 ㅋㅋ

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145).

**Pull request merged**